configs/cortezaproject.json

```diff
@@ -19,5 +19,8 @@
   "conversation_id": [
     "1576826212"
   ],
+  "custom_settings": {
+    "attributesForFaceting": ["version"]
```

Comment:
It seems you also have `language` and `module`, wouldn't you also use them as `attributesForFaceting`?

Reply:
I added those extra two because I thought I would also like to have them, but I'll probably remove them for now

Reply:
Sounds good!